### PR TITLE
Css and js

### PR DESCRIPTION
**Pull request by bibiebliss**

### CODE REVIEW

**Comment by logannc** [comment]
Something weird is going on here. Why does this PR include all previous commits?

**Comment by bibiebliss** [comment]
I don't know... could it be because I tried to pull? I just wanted an updated version on the branch I was working on so I could preview the complete web page.... is this what you mean? If not i don't think I got your question right

**Comment by logannc** [comment]
That shouldnt have done it. If you look at the commits in those PRs, they
have the same hashes that are present on main. So the diff between your
branch and main shouldnt include them.

It makes it very hard to know what i should be reviewing.

On your next PR, note what git commands you use. Im curious about whats
happening.

**Comment by bibiebliss** [comment]
here the last 4 commits are what I remember doing on this branch

**Comment by bibiebliss** [comment]
for the most recent commit (changing fonts) this is a screenshot of how i did it
Screen Shot 2021-05-30 at 11 05 14 PM

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -1,9 +1,197 @@
...
-#greeting-container {
-  margin-top: 20px;
-}
+html, body{
+    height:100%;
+    width:100%;
+    }
+    
+    /* Navigation CSS */
+    #navbar{
```
Comment (logannc) [inline portfolio/src/main/webapp/style.css]: You might want to format this file too.

**Review by logannc — APPROVED** [review]

**Pull request merged**